.betterer.results

```diff
@@ -13330,7 +13330,25 @@ exports[`no undocumented stories`] = {
     ],
     "public/app/core/components/PageHeader/PageHeader.tsx:952684291": [
       [65, 22, 22, "Do not use any type assertions.", "4054086100"],
+<<<<<<< HEAD
       [104, 34, 21, "Do not use any type assertions.", "2401004021"]
+||||||| parent of 73f6df211f (Moving user profile routes to navId)
+      [102, 34, 21, "Do not use any type assertions.", "2401004021"]
+    ],
+    "public/app/core/components/PageNew/PageTabs.tsx:3261576880": [
+      [23, 22, 22, "Do not use any type assertions.", "4054086100"]
+    ],
+    "public/app/core/components/PageNew/SectionNav.tsx:3562461661": [
+      [20, 34, 21, "Do not use any type assertions.", "2401004021"]
+=======
+      [102, 34, 21, "Do not use any type assertions.", "2401004021"]
+    ],
+    "public/app/core/components/PageNew/PageTabs.tsx:3261576880": [
+      [23, 22, 22, "Do not use any type assertions.", "4054086100"]
+    ],
+    "public/app/core/components/PageNew/SectionNav.tsx:1124011774": [
+      [20, 34, 21, "Do not use any type assertions.", "2401004021"]
+>>>>>>> 73f6df211f (Moving user profile routes to navId)
```

Comment:
`yarn betterer:merge; yarn betterer -u`

Reply:
did not work, complained that it could "Unexpected token [ in JSON at position 425" 

Did a manual sync with file in main